vizro-core/src/vizro/models/types.py

```diff
@@ -161,7 +166,7 @@ class capture:
     [custom charts](../user_guides/custom_charts.md).
     """
 
-    def __init__(self, mode: Literal["graph", "action"]):
+    def __init__(self, mode: Literal["graph", "action", "table"]):
```

Comment:
This function needs quite a few other updates now that `table` is an option. If you just search the file for `"graph"` then you should see where docstrings etc. need to be updated.

Reply:
Done